### ISSUE TITLE
LicenseAcceptanceWindow and LicenseFileWindow Accessibility issues

### ISSUE THREAD

**Issue by nkolev92** [NuGet/Home#7452]
from @PatoBeltran  in https://github.com/NuGet/NuGet.Client/pull/2495#issuecomment-434468385
- The accessibility for both the license window and the expanded license acceptance window seems to be broken, I suggest doing the following:
  - [x] Fix any issues that appear in Keros for these windows
  - [x] Make sure you can use narrator to navigate the UI three in both windows and get everything read
  - [x] Look into the live region event to announce to the user that the license acceptance window has expanded. **_Live region is not available in net46_**.

**Comment by PatoBeltran** [comment]
I did an accessibility pass and found a couple of issues/concerns. Here they are:

### UI:
- [x] Link for "View License" is underlined (it shouldn't because it's not consistent with other license links) (In both, details pane and license acceptance)
- [x] Link for "View License" has a weird color when hovered (In both, details pane and license acceptance)
- [x] Should the accept/decline buttons also move to the right limit of the window? - **Mentioned it to the @karann-msft  and he said it's ok like that, as it preserves the context. **

### Keyboard navigation:
- [x] Tab stop on license acceptance window is weird when open. Should it go to the opened license before the buttons?

### Narration:
- [x] Weird "increase zoom", "decrease zoom" narration on license acceptance window. **Known bug being fixed in .NET Framework 4.8**
- [x] License acceptance window. When opening a license and then changing the one that is open, narrator announces the first one, even though the UI has another one. **Reached out to folks for help, more info the below [comment](https://github.com/NuGet/Home/issues/7452#issuecomment-437147024). Not relevant anymore as of [this](https://github.com/NuGet/Home/issues/7452#issuecomment-439145805)**
- [ ] Jaws does not narrate flow document. In license window or license acceptance window. **Seems like a jaws deficiency, as the other 2 readers work fine**
- [x] Screen reader doesn't announce that license acceptance window has been expanded. **Not relevant anymore**

**Comment by nkolev92** [comment]
@PatoBeltran 
The first 2 I already have a fix for I will create the PR (marked them as complete)

I will keep adding the status of each issue by just editing your comment.

**Comment by karann-msft** [comment]
license acceptance might be going away completely, hence didn't want to put a lot of engineering effort into it
https://github.com/NuGet/Home/issues/7439

**Comment by nkolev92** [comment]
Regarding the narration issue with license switching. 

On a click, the context behind the control changes. 
Say the first document is being narrated, when you click on a different license, it will update the document view. 
If you try narrating the whole document again it continues narrating the previous one. 
If you do go to the end of text and try narrating the same thing again, it will then read the updated document. 

Not sure if this is a WPF/Narrator/User Error yet.

**Comment by nkolev92** [comment]
The keyboard navigation will be fixed too.

**Comment by nkolev92** [comment]
After some discussions we decided to a simple modal window instead of expanding the license acceptance window because of the plethora of accessibility issues we have with the current approach. 

Updating Pato's comment with the correct information.
